### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by gu-hu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# TaigaMobile  
-<a href='https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' width=300/></a>  
+# TaigaMobile
+<a href='https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' width=300/></a>
+
+This is the **unofficial** android app for the agile project management system [taiga.io](https://www.taiga.io/). It was built with [Jetpack Compose](https://developer.android.com/jetpack/compose), featuring Material You with dynamic colors. The project is still under development and new features will be added in the future.
```
Comment (EugeneTheDev) [inline README.md:4]: `This is the` - i'm not sure that we need `the` here

File: README.md [hunk 2]
```diff
@@ -39,10 +39,10 @@ This is **unofficial** android app for [taiga.io](https://www.taiga.io/) (agile
...
-Probably sucks. I'm not very good at designing UI, but I'm doing my best. So, if you have any suggestions, please tell me. I'll be really glad to receive any feedback.  
+Probably sucks. I'm not very good at designing UI, but I'm doing my best. So, if you have any suggestions, please tell me. I'll be really glad to receive any feedback.
 
 ## Download
-Check out app on [Google Play](https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github) or go to [Releases](https://github.com/EugeneTheDev/TaigaMobile/releases) page to download apk.
+Check out the app on [Google Play](https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github) or go to the [Releases page](https://github.com/EugeneTheDev/TaigaMobile/releases) to download the latest apk.
 
 ## Issues
-Please consider telling about problems or suggestions on [Issues](https://github.com/EugeneTheDev/TaigaMobile/issues) page.
+If you have any problems or suggestions, please post an [Issue](https://github.com/EugeneTheDev/TaigaMobile/issues).
```
Comment (EugeneTheDev) [inline README.md:48]: `please post an issue` - I think the better one is `submit an issue`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -22,7 +22,7 @@
     <string name="login_error_message">Error during login. Check credentials or connection</string>
 
     <string name="login_alert_title">Unencrypted connection</string>
-    <string name="login_alert_text">You are going to use unencrypted connection. Are you sure?</string>
+    <string name="login_alert_text">You are going to use an unencrypted connection. Are you sure?</string>
```
Comment (EugeneTheDev) [inline app/src/main/res/values/strings.xml:25]: I'm not sure if `an` is needed here

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -120,22 +120,22 @@
...
-    <string name="unlink_epic_text">Are you sure you want to unlink epic?</string>
+    <string name="unlink_epic_text">Are you sure you want to unlink this epic?</string>
 
     <string name="remove_user_title">Remove user</string>
-    <string name="remove_user_text">Are you sure you want to remove user?</string>
+    <string name="remove_user_text">Are you sure you want to remove this user?</string>
 
     <string name="delete_task_title">Delete task</string>
-    <string name="delete_task_text">Are you sure you want to delete task?</string>
+    <string name="delete_task_text">Are you sure you want to delete thist ask?</string>
```
Comment (EugeneTheDev) [inline app/src/main/res/values/strings.xml:129]: Typo here

**Review by EugeneTheDev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gu-hu) [reply README.md:4]: English grammar requires an article here. The "the" is needed for the sentence to be grammatically correct.

File: README.md (hunk 2 above)
Comment (gu-hu) [reply README.md:48]: done

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (gu-hu) [reply app/src/main/res/values/strings.xml:129]: done

**Review by EugeneTheDev — APPROVED** [review]
LGTM

**Pull request merged**